conda_build/build.py

```diff
@@ -1140,3 +1138,7 @@ def is_package_built(metadata, config):
     # will be empty if none found, and evalute to False
     package_exists = [url for url in urls if url + '::' + metadata.pkg_fn() in index]
     return package_exists or metadata.pkg_fn() in index
+
+
+def is_noarch_python(build_noarch):
+    return str(build_noarch).lower() == "python"
```

Comment:
Good start.  I think you can make it even shorter by passing ```m``` directly, and then putting the ```get_value``` inside this function.